packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -458,7 +474,7 @@ class _RenderCupertinoSwitch extends RenderConstrainedBox {
     config.isToggled = _value;
   }
 
-  final CupertinoThumbPainter _thumbPainter = CupertinoThumbPainter();
+  final CupertinoThumbPainter _thumbPainter = const CupertinoThumbPainter.switchThumb();
```

Comment:
Interestingly the analyzer doesn't seem to have an opinion about the `const`.

Reply:
Weird, I'm curious as to why.  Definitely better to err on the side of including the const though.

Reply:
Why not just substitute `const CupertinoThumbPainter.switchThumb();` for _thumbPainter below?